counsel.el

```diff
@@ -1695,6 +1700,17 @@ Skip some dotfiles unless `ivy-text' requires them."
 (defvar counsel-find-file-speedup-remote t
   "Speed up opening remote files by disabling `find-file-hook' for them.")
 
+(defun counsel--preselect-file ()
+  "Chooses the file to preselect in find-file like completions depending on customizations"
```

Comment:
How about something like

```el
  "Return candidate to preselect during filename completion.
The preselect behaviour can be customized via user options
`counsel-find-file-at-point' and
`counsel-preselect-current-file', which see."
```

Reply:
My suggestion is basically to try and follow the conventions listed in [`(elisp) Documentation Tips`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html).